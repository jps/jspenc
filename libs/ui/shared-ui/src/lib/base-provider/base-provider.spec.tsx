import { render } from '@testing-library/react';

import BaseProvider from './base-provider';

describe('BaseProvider', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<BaseProvider><span>Content is passed through</span></BaseProvider>);
    expect(baseElement).toMatchSnapshot();
  });
});
