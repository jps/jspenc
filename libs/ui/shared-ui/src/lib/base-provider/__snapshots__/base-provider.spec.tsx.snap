// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseProvider should render successfully 1`] = `
<body>
  <div>
    <span>
      Content is passed through
    </span>
  </div>
</body>
`;
